pages/validator/external-chains/optimism.md

```diff
@@ -0,0 +1,440 @@
+# Optimism
+-----------
+
+import Markdown from 'markdown-to-jsx'
+import Tabs from '../../../components/tabs'
+import CodeBlock from '../../../components/code-block'
+import { NextraTab, NextraTabs } from 'nextra-theme-docs'
+
+Instructions to set up your Optimism node.
+
+## Requirements
+
+- [Setup your Axelar validator](/validator/setup)
+- Minimum hardware requirements: 8 AWS vCPU+, 16GB RAM, 80GB+ free storage space.
+- MacOS or Ubuntu 20.04 (tested on 20.04)
+- [Official Documentation](https://community.optimism.io/docs/developers/build/run-a-node/#)
+
+
+## Prerequisites
+```bash
+sudo apt-get install jq -y
+snap install docker 
+```
+
+## Clone Repo
+<Tabs tabs={[
+{
+title: "Mainnet",
+content: <CodeBlock language="bash">
+{`
+git clone https://github.com/smartcontracts/simple-optimism-node.git
+cd simple-optimism-node
+cp .env.example .env
+`}
+</CodeBlock>
+},
+{
+title: "Testnet",
+content: <CodeBlock language="bash">
+{`
+git clone https://github.com/ethereum-optimism/simple-optimism-node.git
+cd simple-optimism-node
+cp .env.example .env
+`}
+</CodeBlock>
+}
+]} />
+
+## Set environment variables
+
+<Tabs tabs={[
+{
+title: "Mainnet",
+content: <CodeBlock language="bash">
+{`
+NETWORK_NAME=mainnet
+NODE_TYPE=full
+# can be l1 or l2
+SYNC_SOURCE=
+#  l2 node to check blocks against
+HEALTHCHECK__REFERENCE_RPC_PROVIDER=
```

Comment:
we'll only add optimism support on mainnet after bedrock upgrade there, so mainnet docs can be similar with a note that bedrock isn't active yet.

Reply:
ahhhh wish I knew that earlier, I had to create so many tabs in so many place just because the testnet was on bedrock and mainnet wasn't haha. So I guess we can remove all the tabs then, we would need just one tab for NETWORK_NAME=mainnet and that's it I guess.

But I think we might still have to revisit the instructions once they upgrade to bedrock on mainnet as well.

Reply:
sorry, should have been more clear. original plan was to add it right away, but the bedrock upgrade coincided with our integration haha.

Reply:
I see, no worries